### DISCUSSION TITLE
Modo mantenimiento para Blogger

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#36] [Announcements]
# Modo mantenimiento para Blogger

Cómo poner tu blog de Blogger en modo mantenimiento, para informar a tus usuarios de los cambios que estás realizando.

https://www.zkreations.com/2024/06/modo-mantenimiento-para-blogger.html

**Comment by ximosa** [comment]
Me encanta que sigas dando soporte a blogger, ya que parece que esta pasado de moda y para mi es una joya, ante todo gracias por el post.
He creado un pequeño proyecto en blogger que esta en desarrollo que usa emailjs y la opción de publicar por correo electronico de blogger, puedes probar aqui a ver que te parece, https://video-listado.blogspot.com/, tengo otros proyectos como registro de firebase integrado en blogger, https://usuario.expertowordpress.org/, echale un ojo a ver si podemos colaborar de alguna manera para darle un empujon a blogger

**Reply by danieIabel** [reply]
Que tal amigo. Me encanta Blogger, me gustaría que Google le pusiera mas atención, pienso que si hacemos la comunidad mas grande, eso seria posible. Te invito a mi grupo en telegram donde también compartimos mucho sobre Blogger. Tus ideas me parecen geniales: https://t.me/+xv58NoUx3Ic0ODUx

**Comment by DerickInnovaDigital** [comment]
La verdad me gusta mucho esa función, y esta super nitida, pero al momento de realizar los pasos mencionado en mi blog, me presentaron varios errores, y la verdad no se si es la plantilla Bison, que estoy usando que tiene algo desconfigurado, pero al momento de insertar los 2 código en la head y en body. Al darle guardar me presenta lo siguiente errores:

1- org.xml.sax.SAXParseException; lineNumber: 2865; columnNumber: 5; The element type "b:if" must be terminated by the matching end-tag "</b:if>".

2- org.xml.sax.SAXParseException; lineNumber: 3853; columnNumber: 3; The element type "html" must be terminated by the matching end-tag "</html>".

3- org.xml.sax.SAXParseException; lineNumber: 3853; columnNumber: 10; The markup in the document following the root element must be well-formed.

Quiero poner esa función, por que le estoy danto una actualización a mi blog llamado: DerickInnovaDigital, pero no fue posible, debido a esos errores presentado.

**Reply by danieIabel** [reply]
Que tal amigo, el error que te menciona el editor de Blogger significa que no esta colocando adecuadamente la segunda parte del codigo que he compartido en este tutorial. Al parecer la estas posicionando en un lugar que no deberías. ¿Podrias enviar una captura de donde lo estas ingresando?

**Reply by DerickInnovaDigital** [reply]
Con respecto a tu pregunta, si puedo enviarte una captura, si, pero en este chat de comentario no me permite integrar una imagen.

La verdad no entiendo, por que no me quiere funcionar, si me he asegurado pegarlo en los lugares indicados.

**Reply by danieIabel** [reply]
Para publicar una captura en este formulario, solo debes dirigirte a la url de la comunidad, ahí el formulario de comentarios está completo: [https://github.com/zkreations/data/discussions/36](https://github.com/zkreations/data/discussions/36#discussioncomment-10100687)
